client/src/admin/NotificationsPage.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const NotificationsPage = () => (
+  <div className="container--content">
```

Comment:
I can put some temp content in it if you prefer I'd rather do it in a separate branch.